tests/PHPUnit/System/expected/test_CustomEvents_Events.getName_flat__API.getProcessedReport_day.xml

```diff
@@ -617,6 +617,6 @@
 		<nb_events_with_value>14</nb_events_with_value>
 		<sum_event_value>137.96</sum_event_value>
 		<min_event_value>9.66</min_event_value>
-		<max_event_value>52.32</max_event_value>
+		<max_event_value>9.66</max_event_value>
```

Comment:
There must be something wrong here:

`nb_events_with_value`: 14
`min_event_value`: 9.66
`max_event_value`: 9.66
`sum_event_value`: 137.96

Check in case of `min_event_value` = `max_event_value`:
`nb_events_with_value` * `min_event_value` === `sum_event_value`
`nb_events_with_value` * `max_event_value` === `sum_event_value`

- Fails: 14 * 9.66 = 135,24 is smaller than 137.96

Is this some kind of rounding error?

Reply:
Those are actually caused by another problem. The event metrics seem to be summed up as well. Need to set the correct metadata for that to work...